astropy/wcs/src/unit_list_proxy.c

```diff
@@ -4,6 +4,8 @@
 */
 
 #define NO_IMPORT_ARRAY
+#ifndef ASTROPY_WCS_API_H
+#define ASTROPY_WCS_API_H
```

Comment:
Not sure if this is correct. As far as I see this constant should be defined only in [astropy_wcs_api.h](https://github.com/astropy/astropy/blob/master/astropy/wcs/include/astropy_wcs/astropy_wcs_api.h)

Reply:
Yes I am removing this it is not correct